.pre-commit-config.yaml

```diff
@@ -42,14 +42,17 @@ repos:
       - id: mdformat
```

Comment:
How are the translated pages maintained? If auto-generated, the exclusion rule for pre-commit is probably the right one long term, but I only saw a script for modifying the links ([0e7221f/docs/update_translations.py](https://github.com/ultralytics/ultralytics/blob/0e7221fb62191e18e5ec4f7a9fe6d8927a4446c2/docs/update_translations.py))

Reply:
@KyleKing ah don't worry about these, as long as the english pages render correctly then the translations will also work.